spec/requests/group_history_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+describe GroupHistoryController do
+  describe '#show' do
+    let(:group) { create(:group) }
+    let(:user) { create(:user) }
+
+    subject { get "/groups/#{group.id}/history" }
+
+    before { login_as(user) }
+
+    context 'when everything is fine' do
+      before { group.evaluations << create(:evaluation, point_request_status: Evaluation::ACCEPTED) }
+      before { subject }
+
+      it 'works fine' do
```

Comment:
You can write it in one line if you want.
```suggestion
      it ('works fine') { expect(response).to have_http_status(:ok) } 
```
But maybe your way is more readable. :)

Reply:
I wanted to use a shoulda matcher instead of this (and also for the other test), but idk yet why it does not work. I suspect it is only for controller tests, which would make sense, but it is working for this `it { should render_template :show }`.

I mainly added the draft tag because of this.

Reply:
So now it is clear..
https://github.com/thoughtbot/shoulda-matchers#availability-of-rspec-matchers-in-example-groups